ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -303,6 +303,21 @@ public void ensureConsumedPast(final long seqNum, final Duration timeout)
     );
   }
 
+  @Override
+  public void abortCommand(final CommandId commandId) {
+    commandStatusMap.compute(
+        commandId,
+        (k, v) -> {
+          if (v == null) {
+            throw new IllegalStateException(
```

Comment:
We could actually call this before the command is registered right? If something throws between generating the command id and enqueueing?

Reply:
That's true, it is possible.

Ok, now I just log when something is being aborted, but throw no error either way.